admin/admin-model.php

```diff
@@ -227,8 +227,15 @@ public function update_language( $args ) {
 
 		update_option( 'polylang', $this->options );
 
+		$old_description = maybe_unserialize( get_term( (int) $args['lang_id'] )->description );
```

Comment:
Maybe we should check `get_term()` is returning a `WP_Term` object before processing it?
Otherwise PHPStan won't be happy.